setup.py

```diff
@@ -210,7 +212,10 @@ def run(self):
     if sys.platform != 'win32':
         cmake_command.append(source)
         self.spawn(cmake_command)
-        self.spawn(['make'])
+        if PARALLEL:
```

Comment:
I tried using ```msbuild /m:2 libdynd.sln``` (after running ```vcvarsall.bat```), but had a couple of problems. Is that supposed to work?